db/migrate/20220128223158_drop_inventories_table.rb

```diff
@@ -0,0 +1,5 @@
+class DropInventoriesTable < ActiveRecord::Migration[7.0]
+  def change
+    drop_table :inventories
```

Comment:
Is this necessary? :/

Reply:
Yep. :( I created an inventories table before, different from the current one, and I had to drop it.